CMakeLists.txt

```diff
@@ -0,0 +1,201 @@
+cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+project(rubberband VERSION 2.1.1)
+
+option(BUILD_EXECUTABLE "Build the Rubberband executable" OFF)
+option(BUILD_JNI "Build the Java Native Interface (JNI)" OFF)
```

Comment:
I think this should use `cmake_dependent_option` defaulting to ON if the JNI package is found.

Reply:
Actually I'm not sure `cmake_dependent_option` is a good idea. If the package is *not* found, the value is forced of `OFF` and it will just not build the JNI lib without any warning, even if you passed `-DBUILD_JNI=ON`.

Reply:
We could add a message saying the JNI was not found.

Reply:
With `cmake_dependent_option` you can't because the value is forcibly set to `OFF`.

Reply:
`cmake_dependent_option` was just a suggestion for how to implement it. My main point is that it would be nice to build the JNI automatically if the required dependency is found.

Reply:
How about adding `feature_summary` and one `add_feature_info` for every `option`?!